maintenance_plan/tests/test_maintenance_plan.py

```diff
@@ -1,16 +1,21 @@
 # Copyright 2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from datetime import timedelta
-
 from dateutil.relativedelta import relativedelta
+from freezegun import freeze_time
 
 from odoo import _, fields
 
 from .common import TestMaintenancePlanBase
 
 
+@freeze_time("2023-01-31")
 class TestMaintenancePlan(TestMaintenancePlanBase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.today_date = fields.Date.from_string(fields.Date.today())
```

Comment:
Hardcode dates instead of using this and having to check up which day it is:

```suggestion
        cls.today_date = fields.Date.from_string("2023-01-31")
```

and maybe storing this date doesn't have too much sense, but put it directly in asserts.